reverse_engineering/helpers/redshiftHelper.js

```diff
@@ -539,6 +561,7 @@ const getJsonSchema = async (documents, schemaName, tableName) => {
 			properties: getJsonSchemaFromRows(documents, rows)
 		};
 	} catch (err) {
+		helperLogger.log('error', { message: `get json schema error` }, 'get json error');
```

Comment:
transfrom parameters to object and pass logger via parameters, avoid usage of global variables